probe/overlay/weave.go

```diff
@@ -428,11 +434,32 @@ func (w *Weave) addCurrentPeerInfo(latests map[string]string, node report.Node)
 		latests[WeavePluginStatus] = "running"
 		latests[WeavePluginDriver] = "weave"
 	}
+	node = node.AddPrefixTable(WeaveConnectionsTablePrefix, getConnectionsTable(w.statusCache.Router))
 	node = node.WithParents(report.EmptySets.Add(report.Host, report.MakeStringSet(w.hostID)))
 
 	return latests, node
 }
 
+func getConnectionsTable(router weave.Router) map[string]string {
+	const (
+		outboundArrow = "->"
+		inboundArrow  = "<-"
+	)
+	table := make(map[string]string, len(router.Connections))
+	for _, conn := range router.Connections {
+		arrow := inboundArrow
+		if conn.Outbound {
+			arrow = outboundArrow
+		}
+		// TODO: we should probably use a multicolumn table for this
+		//       but there is no mechanism to support it yet.
```

Comment:
How do you mean exactly? I considered using EdgeMetadata and render them in the app but then discarded it.